#request
string mapfile
---

#response
bool success
string message
